Extensions/xkit_patches.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE XKit Patches **//
-//* VERSION 7.4.4 **//
+//* VERSION 7.4.5 **//
```

Comment:
this needs a further version bump now; this will cause a conflict but that's easy enough to resolve with the web interface

```suggestion
//* VERSION 7.4.6 **//
```